.pre-commit-hooks.yaml

```diff
@@ -32,15 +40,15 @@
 
 - id: golint
   name: golint
-  description: Golint is a linter for Go source code.
+  description: Golint is a linter for Go source code
   entry: hooks/golint.sh
   language: script
   files: \.go$
   exclude: vendor\/.*$
 
 - id: yapf
   name: yapf
-  description: yapf (Yet Another Python Formatter) is a python formatter from Google.
+  description: yapf (Yet Another Python Formatter) is a python formatter from Google
   entry: hooks/yapf.sh
   language: script
   files: \.py$
```

Comment:
Why remove all the periods?

Reply:
For consistency. I don't mind to put periods everywhere ^_^